source/extensions/omni.isaac.lab/config/extension.toml

```diff
@@ -24,7 +24,8 @@ requirements = [
     "toml",
     "hidapi",
     "gymnasium==0.29.0",
-    "trimesh"
+    "trimesh",
+    "torch"
```

Comment:
This is not what I meant. You need to modify the version named: https://github.com/isaac-sim/IsaacLab/blob/5f6a984819566ebfa8a0635f3168fdfb01e4537e/source/extensions/omni.isaac.lab/config/extension.toml#L4